src/types/ERC725JSONSchema.ts

```diff
@@ -158,10 +160,12 @@ export function isValidValueType(
  */
 export interface ERC725JSONSchema {
   name: string; // Describes the name of the key, SHOULD be composed of the Standards name + sub type. e.g: LSP2Name
+  nonDynamicName?: string; // Describes the name of the key, SHOULD be composed of the Standards name + sub type. e.g: LSP2Name without dynamic part
```

Comment:
Suggest moving all optional properties at the bottom:

```ts
export interface ERC725JSONSchema {
  name: string; // Describes the name of the key, SHOULD be composed of the Standards name + sub type. e.g: LSP2Name
    key: string; // The keccak256 hash of the name. This is the actual key that MUST be retrievable via ERC725Y.getData(bytes32 key)
  keyType: ERC725JSONSchemaKeyType | string; // Types that determine how the values should be interpreted.
  valueContent: ERC725JSONSchemaValueContent | string; // string holds '0x1345ABCD...' If the value content are specific bytes, than the returned value is expected to equal those bytes.
  valueType: ERC725JSONSchemaValueType | string; // The type of the value. This is used to determine how the value should be encoded / decode (`string` for tuples and CompactBytesArray).
  dynamicKeyParts?: DynamicKeyParts;
  nonDynamicName?: string; // Describes the name of the key, SHOULD be composed of the Standards name + sub type. e.g: LSP2Name without dynamic part
}
```

Reply:
ok.